harvester/src/test/java/com/github/reflectoring/infiniboard/harvester/source/url/UrlSourceRetrieveJobTest.java

```diff
@@ -0,0 +1,12 @@
+package com.github.reflectoring.infiniboard.harvester.source.url;
+
+import org.junit.Test;
+
+public class UrlSourceRetrieveJobTest {
+
+    @Test
+    public void saveToDB() {
```

Comment:
This test seems to not do anything. Please remove or complete the test before merging.